api/types/payloads.go

```diff
@@ -85,3 +85,12 @@ type UpdateContactRequest struct {
 	Email     string `json:"email" validate:"required,email"`
 	Phone     string `json:"phone" validate:"required"`
 }
+
+type OrderByInvoiceResponse struct {
```

Comment:
You can remove this and just use the `Order` entity

Reply:
Fixed!